01-data-structures/03-linked-lists/linked_list_spec.rb

```diff
@@ -10,76 +10,45 @@
   let(:n4) { Node.new("Vladimir") }
   let(:linked_list) { LinkedList.new }
 
-  def pusher
+  def push_test_linkedlist
```

Comment:
I would name this push_three_items
because that's what it's doing
everything else is redundant :P